coffee_httpd/src/main.rs

```diff
@@ -15,7 +15,9 @@ async fn main() {
         println!("{err}");
     }
     let mut coffee = coffee.unwrap();
-    coffee.setup(&cmd.cln_path).await.unwrap();
+    if let Err(err) = coffee.setup(&cmd.cln_path).await {
+        println!("{err}");
+    }
```

Comment:
do you think that we should return the error to the caller? like `async main() -> Result<()>`